reactful-numbers/src/app/components/ButtonsBox.js

```diff
@@ -1,67 +1,168 @@
 import React from "react";
 import Button from "./Button";
-function ButtonsBox() {
+
+function ButtonsBox({
```

Comment:
These are all the props passed from the CalculatorBox component where all the state is stored